### PR TITLE
Rename variables

### PR DESCRIPTION
**Pull request by shilpiBose29**
Prefixing exported variables with DOSARRAY_ to ensure there are no namespace conflicts

### CODE REVIEW

File: experiments/dosarray_slowloris_experiment.sh [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#/bin/sh -e
```
Comment (niksu) [inline experiments/dosarray_slowloris_experiment.sh]: Why was this line removed?

File: experiments/dosarray_multi_experiment_dedos.sh [hunk 2]
```diff
@@ -17,7 +17,7 @@ then
   echo "Need to configure DoSarray -- could not find dosarray_config.sh at \$DOSARRAY_SCRIPT_DIR/config (${DOSARRAY_SCRIPT_DIR}/config)" >&2
   exit 1
 fi
-source "${DOSARRAY_SCRIPT_DIR}/config/dosarray_config.sh"
```
Comment (niksu) [inline experiments/dosarray_multi_experiment_dedos.sh]: Was this line redundant?

Comment (shilpiBose29) [reply experiments/dosarray_multi_experiment_dedos.sh]: This line wasn't removed. I just added an additional space

File: experiments/dosarray_slowloris_experiment.sh (hunk 1 above)
Comment (shilpiBose29) [reply experiments/dosarray_slowloris_experiment.sh]: Was an accident. Adding it back

File: experiments/dosarray_multi_experiment_dedos.sh (hunk 2 above)
Comment (shilpiBose29) [reply experiments/dosarray_multi_experiment_dedos.sh]: Agreed. I'll add it back
